rmake.py

```diff
@@ -210,6 +216,17 @@ def config_cmd():
     create_dir( os.path.join(build_path, "clients") )
     os.chdir( build_path )
 
+    # compiler
+    if args.use_hipcc_compiler:
+        cmake_options.append(f"-DCMAKE_C_COMPILER={rocm_path}/bin/hipcc")
+        cmake_options.append(f"-DCMAKE_CXX_COMPILER={rocm_path}/bin/hipcc")
+    else:
+        if os.name == "nt":
+            print("Currently forcing hip-clang compiler on Windows.")
```

Comment:
I meant guard out both cases as even the top ones aren't tested on windows (could be a linux only option?), hipcc.bat is not the same as the clang.exe we use in toolchain so seeing that is misleading.  If the build shows it actually is used then that is also fine.  There was even a hipcc.exe at one point but that is in the same case.